Dockerfile

```diff
@@ -10,6 +10,8 @@ FROM php:${PHP_VERSION}-apache AS php_base
 EXPOSE 80
 WORKDIR /app
 
+ARG CURRENT_UID=1000
```

Comment:
Rename the arg DEV_UID and move it just after the "# Add dev user" comment.
It will prevent us rerunning all the previous layer if we need to change it in the future.